README.md

```diff
@@ -1,110 +1,97 @@
-# FLAMEGPU2 Ensemble Experiments
+# FLAMEGPU2 Ensemble Benchmark
 
-This repository contains the figures relating to ensembles shown in the talk *Simulate the World and Everything in It: Complex Systems Simulations with FGPU2* given by Paul Richmond at [GTC 2021](https://www.nvidia.com/en-us/gtc/topics/developer-tools/), along with the code to generate them. The code demonstrates the effect and scaling of the concurrency feature of the FLAMEGPU2 agent-based modelling framework. Please note that generated figures may differ slightly as a result of the stochastic nature of the simulations and hardware differences.
+This repository contains performance benchmarking of a [FLAME GPU 2](https://github.com/FLAMEGPU/FLAMEGPU2) implementation of Boids agent based model running multiple concurrent simulations.
 
-## Figures
+The code demonstrates the effect and scaling of the concurrency feature of the FLAMEGPU2 agent-based modelling framework.
 
-### Small Populations (Brute force)
-
-![Figure 2.1](./results/figures/small--small_pop_brute_force.png)
-*__Small populations brute force__ This figure shows how the average simulation time scales with ensemble size for different population sizes of a brute force messaging boids model.*
-
-Generated using an `NVIDIA Titan V` GPU.
-
-### Small Populations (Spatial)
 
-![Figure 2.1](./results/figures/small--small_pop.png)
-*__Small populations spatial__ This figure shows how the average simulation time scales with ensemble size for different population sizes of a spatial messaging boids model.*
+## Benchmark Description and Results
 
-Generated using an `NVIDIA Titan V` GPU.
+A number of experiments are undertaken within this benchmark. There is a range of raw data in the [`sample/data`](sample/data) directory with a description of the machine configurations used to generate it in each directory.
 
-### Large Populations (Brute force)
+The results below are from the V100 runs on the Bessemer HPC system at the University of Sheffield. Job submission scripts are included in the [`scripts/slurm`]() folder.
 
-![Figure 2.1](./results/figures/large--large_pop_brute_force.png)
-*__Large populations brute force__ This figure shows how the average simulation time scales with ensemble size for different large population sizes of a brute force messaging boids model.*
+### Small Populations (Brute force)
 
-Generated using an `NVIDIA Titan V` GPU.
+This figure shows how the average simulation time scales with ensemble size for different population sizes of a brute force messaging Boids model.
+	+ Population sizes are stepped between 128 and 1024 at intervals of 128
```

Comment:
This Markdown list will not render correctly.

Reply:
fixed